### ISSUE TITLE
about the result of NTIRE 2018 track 2

### ISSUE THREAD

**Issue by Tomlk** [Junshk/CinCGAN-pytorch#20]
about the result of NTIRE 2018 track 2, has somebody test the NTIRE 2018 track 2 valid dataset? I think the paper's result is not true because even the bicubic result and the BM3D+EDSR result is not true.

**Comment by jzsherlock4869** [comment]
@Tomlk  hi there~ I tested the ntire 2018 track 2 (x4 mild) validation set, my result of PSNR for bicubic is aroud 17.56dB (PSNR calculated on RGB colorspace), which is far from the values reported in original paper. I am also wondering the PSNR calculation process the paper used 'cause there are no official code released.

btw, what is your test result (PSNR/SSIM) on bicubic of ntire18track2 val set?
